com.sweethome3d.Desktop.json

```
@@ -0,0 +1,53 @@
+{
+    "app-id" : "com.sweethome3d.Desktop",
+    "branch": "stable",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "19.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "SweetHome3D",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--filesystem=home"
+    ],
+    "modules" : [
+        {
+            "name": "sweethome3d",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir /app/lib/",
+                "cp -r ./ /app/lib/sweethome3d/",
+                "mkdir -p /app/bin/",
+                "ln -s /app/lib/sweethome3d/SweetHome3D /app/bin/",
+                "install -Dm644 SweetHome3DIcon.png /app/share/icons/com.sweethome3d.Desktop.png",
+                "install -Dm644 com.sweethome3d.Desktop.desktop /app/share/applications/com.sweethome3d.Desktop.desktop",
+                "install -Dm644 com.sweethome3d.Desktop.appdata.xml /app/share/appdata/com.sweethome3d.Desktop.appdata.xml",
```

```suggestion
                "install -Dm644 com.sweethome3d.Desktop.metainfo.xml /app/share/metainfo/com.sweethome3d.Desktop.metainfo.xml",
```
The .appdata should be renamed to .metainfo